falcon/routing/compiled.py

```diff
@@ -531,6 +578,25 @@ def _instantiate_converter(self, klass, argstr=None):
         src = '{0}({1})'.format(klass.__name__, argstr)
         return eval(src, {klass.__name__: klass})
 
+    def _compile_and_find(self, path, _return_values, _patterns, _converters, params):
+        """Compiles the router, sets the `_find` attribute and returns its result
```

Comment:
Since this is a private method, probably it doesn't even need a perfect docstring.

But otherwise it would preferably need to start with an imperative (i.e. _Compile_), and the sentence should end with a full stop (period).

Reply:
I added it since it must keep the same interface as `_find` so a bit of documentation is useful, even if only to highlight among other private methods